### DISCUSSION TITLE
How to have something like vscode profiles?

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#3030] [Q&A]
Having configured neovim for multiple languages, I want to be able to configure profiles for each languge .
More context here, also an example will be very helpful
https://www.headway.io/blog/vscode-profiles-optimize-your-coding-environment

**Comment by dpetka2001** [comment]
That's what the Lazyrc Extra is for.

**Reply by rubiin** [reply]
I couldnt find much documenattaion on it

**Reply by dpetka2001** [reply]
Yeah the docs are lacking in this. You should read the PR #2115 and the links that are attached to it for a better understanding. Basically, in your `/lua/config/lazy.lua` you have to import the Extra as the last one after the line `{ import = "plugins" }` and then you should create a `.lazy.lua` file which returns specific `lazy.nvim` plugin specs with the settings you want. This is more like project profiles, rather than language profiles. I don't know what vscode profiles do.
